kaggle/RANZCR/4th_place_solution/utils.py

```diff
@@ -0,0 +1,116 @@
+import os
+import random
+
+import numpy as np
+import torch
+from monai.optimizers.lr_scheduler import WarmupCosineSchedule
+from numba import jit
+from torch import optim
+from torch.utils.data import DataLoader, SequentialSampler
+
+from data.seg_data import CustomDataset
+
+
+def set_seed(seed=1234):
```

Comment:
We should use the monai utility to do this, if needed, we enhance the monai utilities

Reply:
Hi @wyli , do you mean `set_determinism`? this function use fixed `torch.backends.cudnn.deterministic = True` and `torch.backends.cudnn.benchmark = False`, which are not equal to this pipeline.
Shall I add another `set_seed` function in monai utilities?

Reply:
we could use `set_determinism(seed)` and then set `torch.backends.cudnn.deterministic = False`, `torch.backends.cudnn.benchmark = True`. but I don't understand how this achieve deterministic, maybe they prefer speed rather than determinism?